tool/travis.sh

```diff
@@ -22,7 +22,6 @@ function run_coveralls {
 pushd build_test
 pub upgrade
 dartanalyzer --fatal-warnings lib/build_test.dart
-pub run test
```

Comment:
Probably doesn't hurt to leave this in? Even if there are no tests I think it still passes, and that way if we do add tests they will run.